src/components/molecules/Visualizer/Engine/Cesium/index.tsx

```diff
@@ -99,7 +104,39 @@ const Cesium: React.ForwardRefRenderFunction<EngineRef, EngineProps> = (
           {/* remove default double click event */}
           <ScreenSpaceEvent type={ScreenSpaceEventType.LEFT_DOUBLE_CLICK} />
         </ScreenSpaceEventHandler>
-        <Camera onChange={handleCameraMoveEnd} />
+        <ScreenSpaceCameraController
+          maximumZoomDistance={
+            property?.cameraLimiter?.cameraLimitterEnabled
+              ? property.cameraLimiter?.cameraLimitterTargetArea?.height || Number.POSITIVE_INFINITY
+              : Number.POSITIVE_INFINITY
+          }></ScreenSpaceCameraController>
+        <Camera
+          onChange={handleCameraMoveEnd}
+          percentageChanged={property?.cameraLimiter?.cameraLimitterEnabled ? 0.2 : 0.5}
```

Comment:
```suggestion
          percentageChanged={0.2}
```

Camera onChange is already throttled so percentageChanged can be fixed

Reply:
Sure will do, I thought it boosts the performance a little bit, even if throttle used.